### DISCUSSION TITLE
Keybind to Ignore autocompletion

### DISCUSSION THREAD

**Discussion by NitroSniper** [excalith/excalith-start-page#48] [Ideas]
### Why is this needed (example):
If I search for "advent" with "advent of code" on the homepage, the autocomplete will prioritize "advent of code.", Sometimes I don't want this feature getting in the way and I just want to search for "advent".

### Feature:
Have a keybind like CTRL-ENTER to ignore the autocompletion and just search whatever is in the buffer.

**Comment by excalith** [comment]
Hey @NitroSniper, thanks for the suggestion! It is indeed a nice QOL improvement!

Normally I would avoid such conflicts using a space character to prevent auto-selection, howerver CTRL+Enter surely feels and sounds cooler! I will check your PR as soon as possible. Cheers!

This discussion is related to PR #49
